Jenkinsfile

```diff
@@ -1,49 +1,20 @@
 #!groovy
-@Library('waluigi@v5.0.0') _
-
-standardProperties()
-
-node("primary") {
-  echo "Clean workspace"
-  cleanWs()
-
-  stage("checkout") {
-    checkout localBranch(scm)
-  }
-
-  stage("dependencies") {
-    yarnInstall()
-  }
-
-  stage("stamp") {
-    sh "yarn beehive-flow stamp"
-  }
-
-  stage("build") {
-    sh "yarn build"
-  }
-
-  stage("lint") {
-    sh "yarn lint"
-  }
-
-  bedrockBrowsers()
-
-  stage("update storybook") {
-    def status = beehiveFlowStatus();
-    if (status.branchState == 'releaseReady' && status.isLatest) {
-      sshagent (credentials: ['jenkins2-github']) {
-        sh 'yarn deploy-storybook'
+@Library('waluigi@v6.0.0-rc') _
+
+beehiveFlowBuild(
+  test: {
+    bedrockBrowsers()
+  },
+  customSteps: {
+    stage("update storybook") {
```

Comment:
Given that tests do not stop the build now this will need a guard to ensure it only runs when the tests have succeed.

- [ ] Guard against running when tests have failed

Reply:
This is no longer necessary as we've changed it to never run these steps if the build is unstable.